src/tests/fixtures/next.config.js-with-function.js

```diff
@@ -1,9 +1,8 @@
-const { PHASE_PRODUCTION_BUILD } = require('next/constants')
```

Comment:
didnt want resolveNextModule here?

Reply:
Nah. For a test fixture it was going to be a big pain to pass-through all the config stuff